### PR TITLE
[D147] Filter Features

### PR DESCRIPTION
**Pull request by tienhoah**

### CODE REVIEW

File: src/apis/choices.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export default axios.create({
+  baseURL: `${process.env.REACT_APP_MERMAID_API_URL}/v1/`
```
Comment (brianbancroft) [inline src/apis/choices.js]: Does REACT_APP_MERMAID_API_URL and REACT_APP_SUMMARY_API_URL come from the same server?

File: src/apis/summary.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import axios from 'axios';
 
 export default axios.create({
-  baseURL: `${process.env.REACT_APP_API_URL}/v1/`
+  baseURL: `${process.env.REACT_APP_SUMMARY_API_URL}/v1/`
```
Comment (brianbancroft) [inline src/apis/summary.js]: Does REACT_APP_MERMAID_API_URL and REACT_APP_SUMMARY_API_URL come from the same server?

File: .circleci/config.yml [hunk 3]
```diff
@@ -42,7 +42,7 @@ jobs:
       - checkout
       - setup_remote_docker
       - *install_dependencies
-      - run:
+      - run: 
           name: Build
           command: |
               export REACT_APP_API_URL=https://summary-api.datamermaid.org;npm run build
```
Comment (brianbancroft) [inline .circleci/config.yml]: I'm also noticing that this doesn't match the name of the two environment variables you've generated.

File: src/components/AutocompleteFilter.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        filterSelectedOptions
+        renderInput={params => <TextField {...params} variant="outlined" label={label} fullWidth />}
+        renderOption={(option, { inputValue }) => {
+          const matches = match(option, inputValue);
+          const parts = parse(option, matches);
+
+          return (
+            <div>
+              {parts.map((part, index) => (
+                <span key={index} style={{ fontWeight: part.highlight ? 700 : 400 }}>
```
Comment (brianbancroft) [inline src/components/AutocompleteFilter.js]: Suggesting that you don't use just "index" to key a iterated element, as that index key might be used elsewhere, and a change might trigger re-renders in unrelated bits. I tend to use something such as "`${index}-${ some string}`" as one of my patterns in general.

File: .circleci/config.yml (hunk 3 above)
Comment (tienhoah) [reply .circleci/config.yml]: I run a local server to have better control on the api

**Comment by tienhoah** [comment]
closed #147

**Pull request merged**